README.md

```diff
@@ -1,12 +1,20 @@
-# Next CI Helper Scripts
+# Next CI Helper Scripts [![CircleCI](https://circleci.com/gh/Financial-Times/next-ci-shared-helpers.svg?style=svg&circle-token=33bcf2eb98fe2e875cc66de93d7e4a50369c952d)](https://github.com/Financial-Times/next-ci-shared-helpers)
 
-## Development
+This repo contains a set of common bash scripts to be used in the delivery pipeline.
+
+
+## Installation
 
-Once you've cloned this repo, run `make install` before you start development.
-This will install local git hooks for the repo to help prevent you accidentally
-committing secrets into git.
+```sh
+git clone git@github.com:Financial-Times/next-ci-shared-helpers.git
+cd next-ci-shared-helpers
+make install
+```
+
+
+## Development
```

Comment:
Did you discover a process that will enable the development process of troubleshooting existing commands and creating new ones?

I wouldn't expect that it is as straightforward as developing code as you would normally, so some guidance might be useful.

Reply:
Good shout! I'm now looking into how to add these helpers project 👍  and will include some examples.

In terms of debugging existing scripts or creating new ones, I'm not too sure what to suggest, most commands require linux so unless you have a vm i cant quite think of how you would do it.

@andygout any suggestions?

Reply:
Nothing off the top of my head as I've not had much experience writing these sorts of scripts, so I ask out of a curiosity of how to go about it.

It looks like @simonplend contributed to the first version of this repo - I wonder if he has any advice… 😉

Reply:
I'll add an example of how to test scripts locally and also how to use these scripts in other repos